src/main/java/com/mercadopago/client/payment/PaymentClient.java

```diff
@@ -41,6 +45,7 @@ public PaymentClient() {
    */
   public PaymentClient(MPHttpClient httpClient) {
     super(httpClient);
+    refundClient = new PaymentRefundClient();
```

Comment:
Aqui não deveria ser passado também o httpClient para utilizar o custom?
```suggestion
    refundClient = new PaymentRefundClient(httpClient);
```

Reply:
Muito bem observado Luquinha, valeu. Feito.